week-1/exercises/paths.js

```diff
@@ -16,3 +16,7 @@ console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
 // Create a variable to store the ext part of the variable
+const dirPart = filePath.slice(0, lastSlashIndex);
```

Comment:
Think about whether the `dir` part of the path should include the `root` part

Reply:
I don't understand Jay, the root part start with quotations marks? "